ecrpc/external_coordinator.proto

```diff
@@ -0,0 +1,83 @@
+syntax = "proto3";
+
+package ecrpc;
+
+import "google/api/annotations.proto";
+
+option go_package = "github.com/ziggie1984/Distributed-Mission-Control-for-LND/ecrpc";
+
+service ExternalCoordinator {
+    // RegisterMissionControl registers mission control data.
+    rpc RegisterMissionControl(RegisterMissionControlRequest) returns (RegisterMissionControlResponse) {
+        option (google.api.http) = {
+            post: "/v1/registermissioncontrol"
+            body: "*"
+        };
+    }
+
+    // QueryAggregatedMissionControl queries aggregated mission control data.
+    rpc QueryAggregatedMissionControl(QueryAggregatedMissionControlRequest) returns (QueryAggregatedMissionControlResponse) {
+        option (google.api.http) = {
+            get: "/v1/queryaggregatedmissioncontrol"
+        };
+    }
+}
+
+// RegisterMissionControlRequest is the request message for registering mission
+// control data.
+message RegisterMissionControlRequest {
+    repeated PairHistory pairs = 1;
+}
+
+// RegisterMissionControlResponse is the response message for registering
+// mission control data.
+message RegisterMissionControlResponse {
```

Comment:
So this is just used for the user experience not for string matching, right ? because we already return `nil` in case if the registration fails otherwise we return empty `RegisterMissionControlResponse`?

https://github.com/mohamedawnallah/Distributed-Mission-Control-for-LND/blob/2a6eea81db8badc02df5535b7999cf34d04e4a7d/handlers.go#L77-L82

Reply:
Yes just for user experience, so that we don't have an empty json object, but at least a string which says e.g. "Successfully registered data ...."